aten/src/ATen/native/Linear.cpp

```diff
@@ -139,6 +174,39 @@ Tensor& linear_out(const Tensor& input, const Tensor& weight, const std::optiona
   return output;
 }
 
+Tensor pack_linear(const Tensor& weight) {
```

Comment:
We have a similar aten function `mkldnn_reorder_linear_weight` (`mkldnn::_reorder_linear_weight`) in `MKLDNNConversions.cpp`. Can we use it instead? Also, since it is specific to mkldnn layout conversions, suggest to move things to `MKLDNNConversions.cpp` too.

Reply:
Hi, thank you for the reviews. I had  a look at the `mkldnn_reorder_linear_weight` function and it doesn't look like we can use it effectively since we'd need to go to `ideep::reorder_if_differ()`. I have moved the `pack_linear` function to `MKLDNNConversions.cpp`.

Reply:
Still, I guess the major difference is the usage of inner product primitive vs. matmul primitive. Can we add a dedicated path for aarch64 in `mkldnn_reorder_linear_weight` for now just like the suggestion I gave for `mkldnn_linear`? We can simplify things later on. cc @yanbing-j

Reply:
Done